### PR TITLE
Fix(admin): Hide fields not relevant for production

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description

Some fields ought to be hidden, see suggestion https://github.com/Clinical-Genomics/Deviations/issues/198

### Changed
- Family view: 
  - hide columns: "created_at", "_cohorts", "synopsis"]
  - hide form fields: "analyses", "cohorts", "links", "synopsis"
- Sample view:
  - hide columns: "age_at_sampling", "invoiced_at", "_phenotype_groups", "_phenotype_terms"
  - hide form fields: "age_at_sampling", "deliveries", "father_links", "flowcells", "invoiced_at", "invoice", "is_external", "_phenotype_groups", "_phenotype_terms", "links", "mother_links"

### Fixed 
- Sample records fast to edit
- Family records fast to edit

### How to prepare for test
- [x] deploy to stage

### How to test family view
- [x] do open family view

### Expected test outcome
- [x] check that all fields mentioned in PR are hidden

### How to test sample view
- [x] do open sample view

### Expected test outcome
- [x] check that all fields mentioned in PR are hidden

- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [x] tests executed by PG
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch
- [ ] Inform to production, AL, VW, production

### CODE REVIEW

File: cg/server/admin.py [hunk 1]
```diff
@@ -323,7 +324,17 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
+        "invoiced_at",
```
Comment (vwirta) [inline cg/server/admin.py:330]: I would like to keep the column for the Invoice (ie the column that is currently the last one on the right). I use it to keep track of what samples have been invoiced. I am not sure what this column is called, could be invoiced_at but I am not sure.

File: cg/server/admin.py [hunk 2]
```diff
@@ -323,7 +324,17 @@ class PoolView(BaseView):
...
+        "loqusdb_id",
+        "organism",
```
Comment (vwirta) [inline cg/server/admin.py:332]: Let's keep this one. It is good for the microbial wgs samples

File: cg/server/admin.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py:330]: You are probably thinking about the "Invoice" column, and we'll keep it visible

File: cg/server/admin.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py:332]: Ok, I we can keep the reference-genome column visible as well I guess they're used together

**Comment by patrikgrenfeldt** [comment]
Local Family test results: (nginx refused stage testing):
image
🆗 "created_at", "_cohorts", "synopsis", "avatar_url" not visible

Local sample test results:
image
🆗  "age_at_sampling", "from_sample", "invoiced_at", "loqusdb_id", "_phenotype_groups", "_phenotype_terms", "time_point" not visible

**Comment by patrikgrenfeldt** [comment]
@emmser Do you have any columns of those suggested to be hidden that production needs to see?

File: cg/server/admin.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py]: I double checked with TN and she said that we should keep both of them

**Comment by AnnaLeinfelt** [comment]
> @emmser Do you have any columns of those suggested to be hidden that production needs to see?

Checking with @moahaegglund also.

File: cg/server/admin.py [hunk 3]
```diff
@@ -323,7 +324,15 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
+        "invoiced_at",
+        "loqusdb_id",
```
Comment (henrikstranneheim) [inline cg/server/admin.py:331]: I would like to keep this one:  "loqusdb_id"

**Comment by Mropat** [comment]
We might not want to display those fields for everyone, but those fields are still being used by production business logic.
Since we dont have a rest api that returns json containing all fields for debugging and testing, but we still need to check those fields if something goes wrong, could you add an alternative way to opt in to view the hidden values? If they are hidden, only way to debug will be via connecting to the database directly, or via cli if such information is added and enabled

**Comment by patrikgrenfeldt** [comment]
> We might not want to display those fields for everyone, but those fields are still being used by production business logic. Since we dont have a rest api that returns json containing all fields for debugging and testing, but we still need to check those fields if something goes wrong, could you add an alternative way to opt in to view the hidden values? If they are hidden, only way to debug will be via connecting to the database directly, or via cli if such information is added and enabled

You should be able to use the command `cg export TABLE IDENTIFIER` to view all data of any table in status-db (e.g. `cg export sample ACC12345`)

File: cg/server/admin.py (hunk 3 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py:331]: Is it OK to have it available via CLI command `cg export sample` or is it needed on an everyday basis?

File: cg/server/admin.py [hunk 4]
```diff
@@ -323,7 +324,15 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
```
Comment (moahaegglund) [inline cg/server/admin.py:329]: I would like to keep from_sample, this is used for downsampled samples and it's good to easily check that it's set for these samples.

File: cg/server/admin.py (hunk 3 above)
Comment (patrikgrenfeldt) [reply cg/server/admin.py]: I re-added it

**Pull request merged**